storage/aptosdb/src/metrics.rs

```diff
@@ -7,7 +7,7 @@ use aptos_metrics::{
 };
 use once_cell::sync::Lazy;
 
-pub static APTOS_STORAGE_LEDGER: Lazy<IntGaugeVec> = Lazy::new(|| {
+pub static LEDGER: Lazy<IntGaugeVec> = Lazy::new(|| {
```

Comment:
Can't believe we've been using this weird name.. 

Let's do "LEDGER_COUNTER"..

Let's keep "aptos_storage_ledger" for now in order not to break the dashboard